releases/8.1/languages/en.php

```diff
@@ -0,0 +1,89 @@
+
+    'array_unpacking_title' => 'Array unpacking support for string-keyed arrays',
+    'array_unpacking_content' => '<p>PHP supported unpacking inside arrays through the spread operator before, but only if the arrays had integer keys. Now it is possible to unpack arrays with string keys too.</p>',
+
+    'performance_title' => 'Performance Improvements',
+    'performance_chart' => '<strong>Symfony Demo App request time</strong><br>
+                25 consecutive runs, 250 requests (sec)<br>
+                (less is better)<br>',
+    'performance_results' => 'The result (relative to PHP 8.0):<br />
+              - 3.5 % WordPress speedup<br />
+              - 27.0% Symfony Demo speedup',
+    'performance_related_functions_title' => 'Performance related features in PHP 8.1:',
+    'performance_jit_arm64' => 'JIT backend for ARM64 (AArch64)',
+    'performance_inheritance_cache' => 'Inheritance cache (avoid relinking classes in each request)',
+    'performance_fast_class_name_resolution' => 'Fast class name resolution (avoid lowercasing and hash lookup)',
+    'performance_timelib_date_improvements' => 'timelib and ext/date performance improvements',
+    'performance_spl' => 'SPL file-system iterators improvements',
+    'performance_serialize_unserialize' => 'serialize/unserialize optimizations',
+    'performance_internal_functions' => 'Some internal functions optimization (get_declared_classes(), explode(), strtr(), strnatcmp(), dechex())',
+    'performance_jit' => 'JIT improvements and fixes',
+
+    'bc_title' => 'Deprecations and backward compatibility breaks',
+    'bc_null_to_not_nullable' => 'Passing null to non-nullable internal function parameters is deprecated.',
+    'bc_return_types' => 'Return types in PHP built-in class methods and deprecation notices.',
+    'bc_returntypewillchange' => 'New <code>#[ReturnTypeWillChange]</code> attribute.',
+    'bc_serializable_deprecated' => '<code>Serializable</code> interface deprecated.',
+    'bc_html_entity_encode_decode' => 'HTML entity en/decode functions process single quotes and substitute by default.',
+    'bc_globals_restrictions' => '<code>$GLOBALS</code> variable restrictions.',
+    'bc_mysqli_exceptions' => 'MySQLi: Default error mode set to exceptions.',
+    'bc_float_to_int_conversion' => 'Implicit incompatible float to int conversion is deprecated.',
+    'bc_finfo_objects' => 'finfo Extension: <code>file_info</code> resource are migrated to existing finfo objects.',
+    'bc_imap_objects' => 'IMAP: imap resources are <code>IMAP\Connection</code> class objects.',
```

Comment:
```suggestion
    'bc_imap_objects' => 'IMAP: imap resources become <code>IMAP\Connection</code> class objects.',
```
Maybe (here and below)? Or following the pattern used below:
```suggestion
    'bc_imap_objects' => 'IMAP: imap resources migrated to <code>IMAP\Connection</code> class objects.',
```